### PR TITLE
Add net.hhoney.rota manifest

### PR DESCRIPTION
**Pull request by cassidyjames**
[ROTA](https://github.com/HarmonyHoney/ROTA) is a fully open source Godot Engine 3.6 game developed by @HarmonyHoney (HHoney Software). I've been working with them to get their game shipped on Flathub. 

It uses the relevant Godot Engine BaseApp and points to the released PCK file, which is equivalent to the game's source code and assets zipped up (technically exported from Godot Engine).

### Please confirm your submission meets all the criteria

- [x] Please describe the application briefly. **Gravity Bending Puzzle Platformer Game**
- [x] The domain used for the application ID is [controlled by the application developer(s)][appid-domain] and the [application id guidelines][appid] are followed.
- [x] I have read and followed all the [Submission requirements][reqs] and the [Submission guide][reqs2].
- [x] I have [built][build] and tested the submission locally.
- [x] I am an author/developer/upstream contributor to the project. If not, I contacted upstream developers about this submission. **Link:** https://github.com/HarmonyHoney/ROTA/issues/6

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[reqs2]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: net.hhoney.rota.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/HarmonyHoney/ROTA.git",
+          "commit": "be542fa2444774fe952ecb22d5056a048399bc25"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/HarmonyHoney/ROTA/releases/download/Flathub/rota-linux.pck",
+          "tag": "Flathub",
```
Comment (bbhtt) [inline net.hhoney.rota.json:28]:
```suggestion
          "url": "https://github.com/HarmonyHoney/ROTA/releases/download/Flathub/rota-linux.pck",
```

File: net.hhoney.rota.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/HarmonyHoney/ROTA.git",
+          "commit": "be542fa2444774fe952ecb22d5056a048399bc25"
+        },
+        {
+          "type": "file",
+          "url": "https://github.com/HarmonyHoney/ROTA/releases/download/Flathub/rota-linux.pck",
```
Comment (bbhtt) [inline net.hhoney.rota.json:27]: this should be versioned.

Comment (bbhtt) [reply net.hhoney.rota.json:27]: Flathub tag is not a versioned tag.

Comment (cassidyjames) [reply net.hhoney.rota.json:27]: Yeah, that stood out to me as well. Their latest tag is `Flathub` but I've recommended they follow semver and will update once they have a new release.

File: net.hhoney.rota.json (hunk 1 above)
Comment (cassidyjames) [reply net.hhoney.rota.json:28]: Whoops, yeah, the tag was supposed to be in the `git` source (just for convenience).

**Comment by bbhtt** [comment]
bot, build net.hhoney.rota

File: net.hhoney.rota.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
+{
+  "id": "net.hhoney.rota",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "base": "org.godotengine.godot.BaseApp",
+  "base-version": "3.6",
```
Comment (hfiguiere) [inline net.hhoney.rota.json:6]: this is likely wrong. I don't think the 3.6 version is built for `24.08`

Comment (hfiguiere) [reply net.hhoney.rota.json:6]: just forget all of this. it's all good. I did mis-read the history.

**Pull request closed without merging**